docs/proposals/tinyms-support.md

```diff
@@ -0,0 +1,144 @@
+- [Implement Sedna Python SDK support TinyMS](#mplement Sedna Python SDK support TinyMS)
```

Comment:
It shows error link when preview. Please fix it to make it preview correctly.
![image](https://user-images.githubusercontent.com/1785240/195823033-b4b3b783-ad6b-44f1-841d-d14b9f38b46b.png)

Reply:
It seems like that IDE is a little different from github in markdown. 
I have fixed this error.

Reply:
The index looks fine to me now